hikari/guilds.py

```diff
@@ -263,12 +263,14 @@ class GuildWidget:
     is_enabled: bool = attr.field(repr=True)
     """Whether this embed is enabled."""
 
-    async def fetch_channel(self) -> channels_.PartialChannel:
+    async def fetch_channel(self) -> typing.Optional[channels_.PartialChannel]:
         """Fetch the widget channel.
 
+        This will be `builtins.None` if not set.
+
         Returns
         -------
-        hikari.channels.PartialChannel
+        typing.Optional[hikari.channels.PartialChannel]
```

Comment:
Should I type this as `channels.GuildTextChannel` since guild widgets can only be set to a text channel?

Reply:
They can actually be set to any channel (including voice). Best to keep it as PartialChannel

Reply:
Ah, my bad :)